[OTLP] Update mapping for OTLP Histograms (#17550)
DataDog/documentation: content/en/metrics/open_telemetry/otlp_metric_types.md

[Read more about distributions][1] to understand how to configure further aggregations.

> Alternatively, if you are using the `counters` mode and enabling the `send_aggregation_metrics` flag, the following metrics would be reported if the histogram bucket boundaries are set to `[-inf, 2, inf]`:

Reviewer: ```suggestion
Alternatively, if you are using the `counters` mode, the `send_aggregation_metrics` flag is enabled, and the histogram bucket boundaries are set to `[-inf, 2, inf]`, the following metrics are reported:
```
